tests/php/test-amp-form-sanitizer.php

```diff
@@ -262,7 +262,9 @@ public function test_converter( $source, $expected = null, $args = [], $expected
 		}
 
 		// Normalize across different testing environments where WP_TESTS_DOMAIN varies.
-		$current_origin = '//' . WP_TESTS_DOMAIN;
+		$tests_domain   = WP_TESTS_DOMAIN;
+		$tests_domain   = strtok( $tests_domain, ':' ); // In wp-env, the WP_TESTS_DOMAIN constant can erroneously include the port: 'localhost:8889'.
```

Comment:
Relatedly, I previously reported https://github.com/WordPress/gutenberg/issues/41038 and fixed via https://github.com/WordPress/gutenberg/pull/41039 an issue where the URL scheme was included in the `WP_TESTS_DOMAIN` (e.g. `http://localhost`). Apparently the port number is now being added.

Reply:
`AMP_Form_Sanitizer_Test` tests were passing for some time in wp-env.  Seems like this has been resurfaced again.